include/bluetooth/mesh/gen_prop_srv.h

```diff
@@ -38,22 +38,14 @@ struct bt_mesh_prop_srv;
 	{                                                                      \
 		.get = _get, .set = _set, .properties = _properties,           \
 		.property_count = _property_count,                             \
-		.pub = { .update = _bt_mesh_prop_srv_update_handler,           \
-			 .msg = NET_BUF_SIMPLE(                                \
-				 BT_MESH_PROP_MSG_MAXLEN(_property_count)) },  \
 	}
 
 /** @def BT_MESH_PROP_SRV_USER_INIT
  *
  * @brief Initialization parameters for a @ref bt_mesh_prop_srv acting as a
  * Generic User Property Server.
  */
-#define BT_MESH_PROP_SRV_USER_INIT()                                           \
-	{                                                                      \
-		.pub = { .update = _bt_mesh_prop_srv_update_handler,           \
-			 .msg = NET_BUF_SIMPLE(BT_MESH_PROP_MSG_MAXLEN(        \
-				 CONFIG_BT_MESH_PROP_MAXCOUNT)) },             \
-	}
+#define BT_MESH_PROP_SRV_USER_INIT() {}
```

Comment:
Me and Ingar had some discussion on this the last week, and because of a bug with the Scene server entry handling, it looks like we'll need to create init macros for all models that have scene data. 
I know both you and Ingar would prefer to keep this consistent, so I'll add init macros for all models as part of that change. I didn't want to remove this macro from the API if it's going to get added back in later anyway.